wandb/sdk/wandb_run.py

```diff
@@ -1010,134 +1005,118 @@ def job_type(self) -> str:
         return self._settings.run_job_type or ""
 
     def project_name(self) -> str:
-        # TODO: deprecate this in favor of project
-        return self._settings.project or ""
+        """Name of the W&B project associated with the run.
+
+        Note: this method is deprecated and will be removed in a future release.
+        Please use `run.project` instead.
+        """
+        deprecate.deprecate(
+            field_name=deprecate.Deprecated.run__project_name,
+            warning_message=(
+                "The project_name method is deprecated and will be removed in a"
+                " future release. Please use `run.project` instead."
+            ),
+        )
+        return self.project
 
     @property
     @_run_decorator._attach
     def project(self) -> str:
         """Name of the W&B project associated with the run."""
-        return self.project_name()
-
-    @_run_decorator._noop_on_finish()
-    @_run_decorator._attach
-    def log_code(
-        self,
-        root: str | None = ".",
-        name: str | None = None,
-        include_fn: Callable[[str, str], bool]
-        | Callable[[str], bool] = _is_py_requirements_or_dockerfile,
-        exclude_fn: Callable[[str, str], bool]
-        | Callable[[str], bool] = filenames.exclude_wandb_fn,
-    ) -> Artifact | None:
-        """Save the current state of your code to a W&B Artifact.
-
-        By default, it walks the current directory and logs all files that end with `.py`.
-
-        Args:
-            root: The relative (to `os.getcwd()`) or absolute path to recursively find code from.
-            name: (str, optional) The name of our code artifact. By default, we'll name
-                the artifact `source-$PROJECT_ID-$ENTRYPOINT_RELPATH`. There may be scenarios where you want
-                many runs to share the same artifact. Specifying name allows you to achieve that.
-            include_fn: A callable that accepts a file path and (optionally) root path and
-                returns True when it should be included and False otherwise. This
-                defaults to: `lambda path, root: path.endswith(".py")`
-            exclude_fn: A callable that accepts a file path and (optionally) root path and
-                returns `True` when it should be excluded and `False` otherwise. This
-                defaults to a function that excludes all files within `<root>/.wandb/`
-                and `<root>/wandb/` directories.
+        return self._settings.project or ""
 
-        Examples:
-            Basic usage
-            ```python
-            run.log_code()
-            ```
+    def get_project_url(self) -> str | None:
+        """Return the url for the W&B project associated with the run, if there is one.
```

Comment:
```suggestion
        """URL of the W&B project associated with the run, if there is one.
```

Could you make the same edits to the original docstring?